tests/functions/test_update_where.py

```diff
@@ -16,3 +19,20 @@ def test_update_where(dataframe):
         ),
         dataframe.replace("Cambridge", "Durham"),
     )
+
+
+def test_update_where_query():
+    """
```

Comment:
Can you add a period to the end of the docstring and put it on one line instead of three?

Reply:
Sure. Will push in the changes soon